zig.html.markdown

```diff
@@ -588,9 +588,6 @@ const x = switch (direction) {
     .South => true,
 };
 
-
-// Switch statements need exhaustiveness.
-// Won't compile: East and West are missing.
 const x = switch (direction) {
```

Comment:
Can you add another comment instead saying how that one's fine since it's exhaustive? Thanks

Reply:
Sure, sounds good

Reply:
Solved with 5cb021cf3e53373770c598062b8079b6d23c9e64